modules/querying/pages/operators-and-expressions.adoc

```diff
@@ -207,6 +207,23 @@ condition := expr
 Strings are compared based on standard lexicographical ordering: +
 (space) < (digit) < (uppercase_letter) < (lowercase_letter).
 
+=== Comparator Support
+Bitwise accumulators support the standard comparisonOperator := "<" | "< =" | ">" | ">=" | "==" | "!=".
```

Comment:
```suggestion
Bitwise accumulators support the standard xref:querying:operators-and-expressions.adoc#_comparison_operators_and_conditions[comparison operators] := "<" | "< =" | ">" | ">=" | "==" | "!=".
```
Most users will not know what `comparisonOperator` is. So, the sentence should read like regular English, but we put in a hyperlink to point them to the documentation that specifies our comparison operators.